src/themes/custom_theme/custom_theme.js

```diff
@@ -0,0 +1,246 @@
+              Cancel: function() {
+                  $( this ).dialog( 'close' );
+                  $( this ).dialog( "destroy" );
+                  var themeName = local_storage.get("theme");
+                  themeName = themeName && themeName.name;
+                  //Reseting charts to previous theme or default.
+							    if (themeName) {
+							        require(['lib/highstock/themes/' + themeName]);
+							    } else{
+							    	Highcharts.setOptions({});
+							    }
+              }
+            }
+      		});
+
+    		win.dialog('open');
+    		generateData();
+    		createChart();
+      });
+  });
+
+  function generateData() {
+	    // generate an array of random data
+	    var time = (new Date()).getTime(),
+	        i;
+
+	    for (i = -999; i <= 0; i += 1) {
+	        var open = Math.round(Math.random() * 100),
+	            close = open - Math.round(Math.random() * 15),
+	            high = open + Math.round(Math.random() * 15),
+	            low = close - Math.round(Math.random() * 15);
+	        chartData.push([
+	            time + i * 1000 * 60,
+	            open, high, low, close
+	        ]);
+	    }
+  }
+
+  function createChart(){
+  	var chartOptions = {
+			chart: {
+          spacingLeft: 0,
+          marginLeft: 45,  /* disable the auto size labels so the Y axes become aligned */
+          //,plotBackgroundImage: 'images/binary-watermark-logo.svg'
+      },
+      series: [{
+          data: chartData
+      }],
+      navigator: {
+          enabled: true,
+          series: {
+              id: 'navigator'
+          }
+      },
+
+      //This will be updated when 'Settings' button is implemented
+      plotOptions: {
+          candlestick: {
+              lineColor: 'black',
+              color: 'red',
+              upColor: 'green',
+              upLineColor: 'black',
+              shadow: true
+          }
+      },
+
+      title: {
+          //Show name on chart if it is accessed with affiliates = true parameter. In normal webtrader mode, we dont need this title because the dialog will have one
+          text: "Some random index"
+      },
+
+      credits: {
+          href: 'http://www.binary.com',
+          text: 'Binary.com',
+
+      },
+
+      xAxis: {
+          ordinal : false
+      },
+
+      yAxis: [{
+          opposite: false,
+      }],
+
+      rangeSelector: {
+          enabled: false
+      },
+
+      tooltip: {
+          crosshairs: [{
+              width: 2,
+              color: 'red',
+              dashStyle: 'dash'
+          }, {
+              width: 2,
+              color: 'red',
+              dashStyle: 'dash'
+          }],
+          enabled: true,
+          enabledIndicators: true
+      }
+  	};
+  	$("#preview-chart").highcharts('StockChart',chartOptions);
+  }
+
+  function updateChart(){
+  	Highcharts.setOptions(themeConf);
+  	$("#preview-chart").highcharts().destroy();
+  	createChart();
+  }
+
+  function openConfirmDialog(){
```

Comment:
@4p00rv This entire thing is duplicated from previous theme code. Can you think of reusing the code?